### PR TITLE
[Needs conformation] mc-host24.de

### PR DESCRIPTION
**Pull request by hasankayra04**
Looks like mc-host24.de needs you to set your Bedrock address to your server ip.

### CODE REVIEW

File: _data/providers.json [hunk 1]
```diff
@@ -256,7 +256,8 @@
     {
       "name": "MC-HOST24.de",
       "url": "https://mc-host24.de/",
-      "description_template": "default"
+      "description_template": "default",
+      "description": "Make sure your remote address is 'auto' and your bedrock address is your server IP."
```
Comment (onebeastchris) [inline _data/providers.json:260]: What about the port? Similar to the other instructions; please include details about the port - Feel free to copy & adapt instructions from another host.

**Review by onebeastchris — CHANGES_REQUESTED** [review]
Thanks for the PR! Do you have a source (e.g. discord message) you can include; just so it's clear when it changed?

File: _data/providers.json (hunk 1 above)
Comment (onebeastchris) [reply _data/providers.json:260]: Ah, makes sense - but nope, as soon as you want to edit something you basically have to write the whole thing :/

**Comment by hasankayra04** [comment]
> Thanks for the PR! Do you have a source (e.g. discord message) you can include; just so it's clear when it changed?

Well kind of. https://discord.com/channels/613163671870242838/613168464634576897/1105805059490582600

**Review by Jens-Co — APPROVED** [review]
LGTM

**Pull request merged**